src/blocks/share/test/share.cypress.js

```diff
@@ -40,7 +40,7 @@ describe( 'Test CoBlocks Share Block', function() {
 		helpers.viewPage();
 
 		cy.get( '.wp-block-coblocks-social li:first-child .wp-block-coblocks-social__button' )
-			.should( 'have.css', 'background-color', 'rgb(49, 55, 60)' );
+			.should( 'have.css', 'background-color', 'rgb(200, 106, 25)' );
```

Comment:
When I see something twice, I tend to create a variable for it. Here, `rgb(200, 106, 25)`